scripts/buildDocs.sh

```diff
@@ -5,8 +5,28 @@ cd "$(dirname -- "$0")/.."
 
 echo "Changed directory to $(pwd)"
 
+# Build image if not existent
+if [[ "$(docker images -q squidfunk/mkdocs-material:8.5.9 2> /dev/null)" == "" ]]; then
+  bash ./scripts/buildImage.sh
+fi
+
+# Mike configuration
+git config --local user.email "github-actions[bot]@users.noreply.github.com"
```

Comment:
why not using our openui5 bot for this one?

Reply:
Done